source/_components/sensor.uk_transport.markdown

```diff
@@ -37,14 +37,32 @@ sensor:
         destination: WAT
 ```
 
-Configuration variables:
-
-- **app_id** (*Required*): Your application id
-- **app_key** (*Required*): Your application key
-- **queries** array (*Required*): At least one entry required.
-- **mode** (*Required*): One of `bus` or `train`.
-- **origin** (*Required*): Specify the three character long origin station code.
-- **destination** (*Required*): Specify the three character long destination station code.
+{% configuration %}
+app_id:
+  description: Your application ID
+  required: true
+  type: string
+app_key:
+  description: Your application KEY
+  required: true
+  type: string
+queries:
+  description: At least one entry required.
+  required: true
+  type: list
+mode:
```

Comment:
The `mode`, `origin` and `destination` appear to be `keys:` of `queries`.

Reply:
Thnx 😄